src/test/java/api/support/matchers/EventMatchers.java

```diff
@@ -36,9 +53,29 @@
       isItemCheckedInEventType());
   }
 
+  public static Matcher<JsonObject> isValidCheckInLogEvent(JsonObject checkedInLoan) {
+    return allOf(JsonObjectMatcher.allOfPaths(
+      hasJsonPath("eventPayload", allOf(
+        hasJsonPath("logEventType", is("CHECK_IN_EVENT")),
+        hasJsonPath("servicePointId", is(checkedInLoan.getString("checkinServicePointId"))),
+        hasJsonPath("returnDate", is(checkedInLoan.getString("returnDate"))),
+        hasJsonPath("loanId", is(checkedInLoan.getString("id"))),
+        hasJsonPath("isLoanClosed", is(checkedInLoan.getJsonObject("status").getString("name").equals("Closed"))),
+        hasJsonPath("systemReturnDate", is(checkedInLoan.getString("systemReturnDate"))),
+        hasJsonPath("returnDate", is(checkedInLoan.getString("returnDate"))),
+        hasJsonPath("dueDate", is(checkedInLoan.getString("dueDate"))),
+        hasJsonPath("userId", is(checkedInLoan.getString("userId"))),
+        hasJsonPath("itemId", is(checkedInLoan.getString("itemId"))),
+        hasJsonPath("itemBarcode", is(checkedInLoan.getJsonObject("item").getString("barcode"))),
+        hasJsonPath("itemStatusName", is(checkedInLoan.getJsonObject("item").getJsonObject("status").getString("name")))
+      ))),
+      isLogRecordEventType());
+  }
+
   public static Matcher<JsonObject> isValidItemClaimedReturnedEvent(JsonObject loan) {
     return allOf(JsonObjectMatcher.allOfPaths(
       hasJsonPath("eventPayload", allOf(
+        hasJsonPath("userId", is(loan.getString("userId"))),
```

Comment:
Please remove it.

Reply:
Hi @bohdan-suprun, could you please clarify what do we need to remove here?